src/manifest.json

```diff
@@ -9,7 +9,7 @@
             "matches": [
                 "<all_urls>"
             ],
-            "js": ["js/compatibility.js","js/pdf.js","js/pdf.worker.js","js/pdftotext.js","js/bundle.js", "js/blacklist2.js","js/background.js","js/lib/stopwords_res.js", "js/lib/readability.js", "js/relevanttext.js", "js/content.js"]
```

Comment:
yes, thats true but only if you dont build the extension. Bundle.js is important for the history. 

You can see the installation procedure here: https://github.com/WorldBrain/Research-Engine/blob/master/README.md#as-a-developer